### PR TITLE
Source Option and Neon Info

### PR DESCRIPTION
**Pull request by drg101**
- Added option for `source` which links to whatever the source page is for the layer/collection
- Added descriptions for all neon stuff
- Added units to the slider
- Made slider text bold. We should make all of the constraints bold, but I didn't want to run into any merge conflicts with @honeybadgerofdoom 's date branch.

### CODE REVIEW

File: src/js/library/autoMenu.js [hunk 1]
```diff
@@ -48,48 +48,51 @@ export default {
                 //These are hardcoded for now
                 let autoMenuLayer = {};
                 if (metadata.level || metadata.linked) {
-                    autoMenuLayer["group"] = metadata["group"] ? metadata["group"] : "Tract, County, & State Data";
-                    autoMenuLayer["subGroup"] = metadata["subGroup"] ? metadata["subGroup"] : metadata.level === "tract" ? "Miscellaneous Tract" : "Miscellaneous County";
-                    autoMenuLayer["linkedGeometry"] = metadata.linked ? metadata.linked.collection : metadata.level === "tract" ? "tract_geo_140mb_no_2d_index" : "county_geo_30mb_no_2d_index";
-                    autoMenuLayer["joinField"] = metadata.linked ? metadata.linked.field : "GISJOIN";
-                    autoMenuLayer["minZoom"] = metadata.level === "tract" ? 9 : 7;
+                    autoMenuLayer.group = metadata.group ? metadata.group : "Tract, County, & State Data";
```
Comment (pierce-smith1) [inline src/js/library/autoMenu.js:51]: It's really convenient to just change these kinds of things as you find them, but can you save formatting/style changes like these for separate "cleanup" PRs so the diffs aren't intermixed with irrelevant stuff for the current PR?

**Review by pierce-smith1 — APPROVED** [review]
The extra details are great and I love the source links.

**Comment by pierce-smith1** [comment]
Something to look at in another PR: These sliders can get kind of rough - I don't know if this is a bug or these values just have an abnormally large range
image

**Comment by drg101** [comment]
> Something to look at in another PR: These sliders can get kind of rough - I don't know if this is a bug or these values just have an abnormally large range
> image

Yea not sure what's going on here.

**Pull request merged**